lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+module Sisimai::Bite::Email
+  module EZweb
+    # Sisimai::Bite::Email::EZweb parses a bounce email which created by au EZweb.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EZweb.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'       => %r/[<]?(?>postmaster[@]ezweb[.]ne[.]jp)[>]?/i,
+        :'subject'    => %r/\AMail System Error - Returned Mail\z/,
+        :'received'   => %r/\Afrom[ ](?:.+[.])?ezweb[.]ne[.]jp[ ]/,
+        :'message-id' => %r/[@].+[.]ezweb[.]ne[.]jp[>]\z/,
+      }
+      Re1 = {
+        :begin  => %r{\A(?:
+             The[ ]user[(]s[)][ ]
+            |Your[ ]message[ ]
+            |Each[ ]of[ ]the[ ]following
+            |[<][^ ]+[@][^ ]+[>]\z
+            )
+        }x,
+        :rfc822   => %r#\A(?:[-]{50}|Content-Type:[ ]*message/rfc822)#,
+        :boundary => %r/\A__SISIMAI_PSEUDO_BOUNDARY__\z/,
+        :endof    => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->